src/__tests__/context.node.js

```diff
@@ -0,0 +1,61 @@
+/** Copyright (c) 2018 Uber Technologies, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import test from 'tape-cup';
+import * as React from 'react';
+import {createToken, createPlugin} from 'fusion-core';
+import {getSimulator} from 'fusion-test-utils';
+import App from '../index';
+import {serviceContextPlugin, ServiceConsumer, useService} from '../context.js';
+
+test('useService hook', async t => {
+  const TestToken = createToken('test');
+  const TestPlugin = createPlugin({provides: () => 3});
+  let didRender = false;
+  function TestComponent() {
+    const provides = useService(TestToken);
```

Comment:
(See note below.  With current types, it looks like this resolves to `ServiceContextType<Token<any | number>>` right now)